src/Event/Event.php

```diff
@@ -383,6 +384,10 @@ public function updateAttendanceMode(AttendanceMode $attendanceMode): void
         if ($this->attendanceMode !== $attendanceMode->toString()) {
             $this->apply(new AttendanceModeUpdated($this->eventId, $attendanceMode->toString()));
         }
+
+        if (!empty($this->onlineUrl) && $this->attendanceMode === AttendanceMode::offline()->toString()) {
+            $this->apply(new OnlineUrlDeleted($this->eventId));
+        }
```

Comment:
Nit: You could drop the `!empty($this->onlineUrl)` and call `$this->deleteOnlineUrl()` instead of `$this->apply(new OnlineUrlDeleted($this->eventId))`

Reply:
This seems to be the same comment?